README.md

```diff
@@ -170,6 +172,35 @@ You can also use `ca: self-sign` or `ca: local` depending on your certmonger usa
 
 Note that this does *not* work on RHEL/CentOS 7.
 
+#### Generate a new certificate in the role
```

Comment:
I find this a bit nicer, and our role retains more control over what actually happens. So :+1: from me for moving to that approach.

But please let's actually *move* -- I find it confusing to offer "you can do it like this or that" in the README. Which one is better? (from the admin's POV). IMHO we should recommend just one approach.

The `setype:` hack is indeed obsolete. RHEL 8.6 has Cockpit 264, and CentOS 8 has something much newer. So we don't need this any more, neither with the "builtin" role.

Reply:
> I find this a bit nicer, and our role retains more control over what actually happens. So +1 from me for moving to that approach.
> 
> But please let's actually _move_ -- I find it confusing to offer "you can do it like this or that" in the README. Which one is better? (from the admin's POV). IMHO we should recommend just one approach.
> 
> The `setype:` hack is indeed obsolete. RHEL 8.6 has Cockpit 264, and CentOS 8 has something much newer. So we don't need this any more, neither with the "builtin" role.

Hmmm, a good point... Maybe, we can say something like this?
```
### Generate a new certificate

Now you can do it in the cockpit role by providing the cockpit_certificates variable as follows
(in the nicer English, of course ;)
    - name: Install cockpit with Cockpit web server certificate
      include_role:
        name: linux-system-roles.cockpit
      vars:
        cockpit_certificates:
          - name: monger-cockpit
            dns: ['localhost', 'www.example.com']
            ca: ipa
            group: cockpit-ws

If you already have a playbook using the certificate role and the cockpit role separately,
that's supported, too.
    - name: Generate Cockpit web server certificate
      include_role:
        name: linux-system-roles.certificate
      vars:
        certificate_requests:
          - name: /etc/cockpit/ws-certs.d/monger-cockpit
            dns: ['localhost', 'www.example.com']
            ca: ipa
            group: cockpit-ws
```

Reply:
Agreed -- documenting this new way as the primary method, and just giving this a fallback is fine.